cylc/flow/scheduler.py

```diff
@@ -1350,36 +1350,30 @@ def release_tasks_to_run(self) -> bool:
                 pre_prep_tasks.update(self.pool.tasks_to_trigger_now)
                 self.pool.tasks_to_trigger_now = set()
 
-            if self.is_paused:
-                # finish processing preparing tasks
-                pre_prep_tasks.update({
-                    itask for itask in self.pool.get_tasks()
-                    if itask.state(TASK_STATUS_PREPARING)
-                })
-            else:
+            if not self.is_paused:
```

Comment:
This change nukes the bug fix that just went in with #7054 

<img width="600" height="402" alt="Image" src="https://github.com/user-attachments/assets/58788196-7083-45d3-90a4-c477c4793dcd" />